frontend/nuxt.config.ts

```diff
@@ -1,5 +1,13 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
+  extends: ['nuxt-umami'],
+  appConfig: {
+    umami: {
+      id: process.env.UMAMI_ID,
+      host: process.env.UMAMI_HOST,
+      useDirective: true,
+    },
+  },
```

Comment:
Please use Nuxt's [useHead](https://nuxt.com/docs/api/composables/use-head) instead of depending on this module. It's just a one line of `<script>` tag anyway